Makefile

```diff
@@ -115,6 +115,10 @@ integration-test-v2: azuredisk-v2
 e2e-test:
 	go test -v -timeout=0 ./test/e2e ${GINKGO_FLAGS}
 
+.PHONY: e2e-test-v2
+e2e-test-v2:
+	go test -v -timeout=0 -tags azurediskv2 ./test/e2e --temp-use-driver-v2
```

Comment:
The `BUILD_V2` environment variable needs to be set to `1`, otherwise you will end up deploying the V1 driver. ([`suite_test.go`](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/8e9db978bb0296289e497d8eb0e2356f40c1ee1d/test/e2e/suite_test.go#L89) execs `make e2e-bootstrap` to deploy the driver and the BUILD_V2 variable needs to be set correctly.)
```
e2e-test-v2:
    BUILD_V2=1 go test ...
```

Reply:
Ah, good point!